### PR TITLE
Resolve#14 add mail both users

### PR DESCRIPTION
**Pull request by flonou**
Adding a new event to send notification to both the new and old user in case of conflict
Adding custom data to get the name of the other user in the conflict
Adding Glpi\Event include when necessary
Translatinglogs  to english

### CODE REVIEW

**Comment by flonou** [comment]
I think the use Glpi\Event; part is glpi 9.4.0 related ?

**Comment by salleman33** [comment]
> I think the use Glpi\Event; part is glpi 9.4.0 related ?

I think it is useless because you don't use function of class Event in Glpi namespace. 
I don't know why only class Event is in Glpi namespace...

**Comment by flonou** [comment]
I actually had errors without it and we do use Event when logging in the main logs:
``` Event::log($reservation['reservations_id'], "Reservation", 4, "inventory", __('Sending an e-mail'));```
 glpi moved it in a namespace because its too much generic for a class with this name to be in the root namespace. I think it was changed in 9.4

File: inc/task.class.php [hunk 1]
```diff
@@ -96,27 +118,28 @@ public static function checkReservations($task) {
                         WHERE `reservations_id`='" . $reservation->fields["id"] . "'";
                $DB->query($query) or die("error on 'update' into checkReservations conflict : " . $DB->error());
             } else {
-               $task->log("conflit avec la reservation " . $conflict_reservation->fields['id'] . " du materiel " . $item->fields['name'] . " par " . $formatName . " (du " . date("d-m-Y \à H:i:s", strtotime($conflict['begin'])) . " au " . date("d-m-Y \à H:i:s", strtotime($conflict['end']).")"));
-               NotificationEvent::raiseEvent('plugin_reservation_conflict', $conflict_reservation);
+               $task->log("conflit for reservation " . $conflict_reservation->fields['id'] . " ontiem " . $item->fields['name'] . " used by " . $formatName . " (from " . date("d-m-Y \à H:i:s", strtotime($conflict['begin'])) . " to " . date("d-m-Y \à H:i:s", strtotime($conflict['end']).")"));
+               NotificationEvent::raiseEvent('plugin_reservation_conflict_new_user', $conflict_reservation, ['other_user_id' => $res['users_id']]);
```
Comment (salleman33) [inline inc/task.class.php]: I think it's better to switch names : conflict_new_user for the next user (user of the next reservation), and conflict_previous_user for the current reservation user, no ?!

**Pull request merged**